### PR TITLE
Tweaking requirements around IDs

### PR DESCRIPTION
**Pull request by christian-herber-nxp**
... to be unique in flight, incremental wrapping, non-continuous

### CODE REVIEW

**Comment by christian-herber-nxp** [comment]
@jklockars: does this address your input?

**Comment by jklockars** [comment]
Fine with me, but it seems you are missing a word:
The ``id`` values for in-flight offloaded instructions _are_ required to be unique.

By the way, I see that the ID length is part of the interface specification. As long as it matches between the coprocessor and the core that is fine. But one or the other (well, certainly the coprocessor, at least) might have a preferred minimum size due to the number of possible in-flight operations - should it be mentioned how that is handled/documented, perhaps?

File: docs/source/x_ext.rst [hunk 1]
```diff
@@ -300,7 +300,8 @@ Identification
 The six interfaces of CORE-V-XIF all use a signal called ``id``, which serves as a unique identification number for offloaded instructions.
 The same ``id`` value shall be used for all transaction packets on all interfaces that logically relate to the same instruction.
 An ``id`` value can be reused after an earlier instruction related to the same ``id`` value is no longer consider in-flight.
-The ``id`` values for in-flight offloaded instructions are only required to be unique; they are for example not required to be incremental.
+The ``id`` values for in-flight offloaded instructions required to be unique.
```
Comment (Silabs-ArjanB) [inline docs/source/x_ext.rst:303]: required -> are required

**Review by Silabs-ArjanB — CHANGES_REQUESTED** [review]

**Comment by christian-herber-nxp** [comment]
> Fine with me, but it seems you are missing a word: The `id` values for in-flight offloaded instructions _are_ required to be unique.
> 
> By the way, I see that the ID length is part of the interface specification. As long as it matches between the coprocessor and the core that is fine. But one or the other (well, certainly the coprocessor, at least) might have a preferred minimum size due to the number of possible in-flight operations - should it be mentioned how that is handled/documented, perhaps?

fixed in 510637f

File: docs/source/x_ext.rst (hunk 1 above)
Comment (christian-herber-nxp) [reply docs/source/x_ext.rst]: fixed in 510637f

**Pull request merged**